src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -7,8 +7,11 @@
 import * as React from 'react';
 import debounce from 'lodash/debounce';
 import flow from 'lodash/flow';
+import getProp from 'lodash/get';
 import noop from 'lodash/noop';
 import { FormattedMessage } from 'react-intl';
+import { withRouter } from 'react-router-dom';
```

Comment:
I think these lines can be simplified to `import { withRouter, type ContextRouter } from 'react-router-dom'`;